server/src/main/java/com/projects/pms/exception/CustomException.java

```diff
@@ -0,0 +1,11 @@
+package com.projects.pms.exception;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class CustomException {
+    private String errorMessage;
```

Comment:
@SunilSunny698 Does we not require to extend the Exception ?

Reply:
No, Here we created this class for sending an exception or error object to front-end. This object contains errorMessage field which represent the reason for the error.